.gitignore

```diff
@@ -128,6 +128,7 @@ dist
 .tern-port
 
 # Stores VSCode versions used for testing VSCode extensions
+.vscode/
```

Comment:
Do we need this in addition to the line below it?

Reply:
Yep, apparently. `.vscode` seems to be a file, and `.vscode/` (which is what I'm used to adding to gitignore), a folder.

I left the ones below alone because it seemed like a small possibility that some dev somewhere MIGHT have it created by their IDE.

Although, typing this and looking at it now, the comment says "VSCode extensions", which has got nothing to do with anything this repo is about. 

But eh. Err on the side of caution.